web/src/user_topics.ts

```diff
@@ -27,7 +27,7 @@ export type UserTopic = {
     visibility_policy: number;
 };
 
-const all_user_topics = new Map<
+export const all_user_topics = new Map<
```

Comment:
This is an abstraction violation; clients should access these data through functions exported by this module.